pkg/transforms/metrics.go

```diff
@@ -64,7 +64,8 @@ func (mp *MetricsProcessor) ToLineProtocol(ctx interfaces.AppFunctionContext, da
 		metric.Tags = append(metric.Tags, mp.additionalTags...)
 	}
 
-	result := metric.ToLineProtocol()
+	// New line is needed if the resulting metric data is batched and sent in chunks to service like InfluxDB
+	result := metric.ToLineProtocol() + "\n"
```

Comment:
use fmt.Sprintln() to avoid hard coded `\n`? : 
```suggestion
	result := fmt.Sprintln(metric.ToLineProtocol())
```

Reply:
I like it. THX!
Changed.